app/assets/stylesheets/dashboard.scss

```diff
@@ -0,0 +1,31 @@
+// Admin dashboard styles
+
+.dashboard-links div {
+  border: 2px solid black;
+  border-bottom: none;
+  padding: .5em;
+}
+
+.dashboard-links:last-child {
+  border-bottom: 2px solid black;
+}
+
+.dashboard-links a:hover {
+  color: black;
+}
+
+.dashboard-links .fa {
+  padding: 0 .25rem;
+}
+.adminAdd {
+  background: lighten( $med_gray, 10% )
+}
+
+.adminManage {
+  background: lighten( $med_gray, 20%  )
+}
+
+.librarianTasks {
+  background: lighten( $med_gray, 30% );
+}
```

Comment:
The grey is totally not as fun as the colors :( Are you committed to it? I trust your judgement to keep this from being tacky, just don't want to end up looking to corporate and boring.